src/Resources/views/components/images/image.html.twig

```diff
@@ -1,5 +1,36 @@
 {#
-    This Zicht component easily adds lazyloading and responsive image support in 1 file include. 
+    This Zicht component easily adds lazyloading and responsive image support in a few simple steps.
+
+    TLDR:
+    1. npm install lazysizes --save
+    2. import 'lazysizes'; // in main.ts
+    3. @import "../../vendor/zicht/htmldev-bundle/src/Zicht/Bundle/HtmldevBundle/Resources/sass/components/lazyload.scss"; // in main.scss
```

Comment:
Is hier geen namespace o.i.d.? Ziet er buggy uit.

Reply:
Hier zou ik even in moeten duiken om te kijken hoe het werkt met namespaces, in overleg even besloten om het zo te houden.